docs/tutorial/1-overview.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 1. Oracle Service Overview
+---
+<!--import { HomeIcon, AppWindow, CircuitBoard, Layers, Microscope, Grid2x2, CloudSun, ChevronRight } from 'lucide-react';-->
+<!--import { Callout } from 'fumadocs-ui/components/callout';-->
+<!--import { Tab, Tabs } from "fumadocs-ui/components/tabs";-->
+<!--import { Scrollycoding } from "@/components/scrollycoding";-->
+import { link } from "@/components/link.tsx"
+
+<img alt="Start-building" src="/start-building.png" />
+
+In this guide, you will build a simple oracle service that fetches Bitcoin price data from [coinmarketcap.com](https://coinmarketcap.com/api/). This example is built using the [WAVS Foundry Template](https://github.com/Lay3rLabs/wavs-foundry-template), which contains the tools you need to build your own custom service.
+
+The price oracle service example has three basic parts:
+
+1. [A trigger](https://github.com/Lay3rLabs/wavs-foundry-template/tree/v0.3.0-rc1/src/contracts/WavsTrigger.sol): A trigger can be any on-chain event emitted from a contract. This event **triggers** a service to run. In the WAVS Foundry Template, there is a simple trigger contract that stores trigger requests, assigns them unique IDs, and emits an event when a new trigger is added. In this example, the trigger event will pass data pertaining to the ID of an asset for the CoinMarketCap price feed.
+
+2. [A service component](https://github.com/Lay3rLabs/wavs-foundry-template/tree/v0.3.0-rc1/components/eth-price-oracle/src/lib.rs): The service component contains the business logic of a service. It is written in Rust (support for more languages is coming soon), compiled to WASM, and run by operators in the WAVS runtime. In this example, operators will listen for a new trigger event to be emitted and then run the service component off-chain, using the asset ID data from the trigger event as input. The component contains logic to fetch the price of the asset from the CoinMarketCap price feed API, which is then processed and encoded before being sent back on-chain.
```

Comment:
i dont think we get to just relative reference this, but a copy over to the main docs from this repo could be cool and simplify docs writing? this way relative paths work and you really only write within this document

Reply:
too much for initial